app/bt/nativeModule.ts

```diff
@@ -26,6 +27,16 @@ export const subscribeToExposureEvents = (
   );
 };
 
+export const fetchLastExposureDetectionDate = async (): Promise<Dayjs | null> => {
```

Comment:
```suggestion
export const fetchLastExposureDetectionDate = async (): Promise<Posix | null> => {
```

What do you think about keeping posix as the data model and waiting to convert to human readable text in the view layer?